test/ui-test/src/screens/StatusCommunityScreen.py

```diff
@@ -475,41 +474,57 @@ def click_sidebar_option(self, community_sidebar_option:str):
             
-    def verify_option_exists(self, option:str):
-        if option=="Permissions":
-            title = get_obj(CommunitySettingsComponents.PERMISSIONS_BUTTON.value).title
-            verify_text(option, str(title))
-        elif option=="Members":
-            title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
-            verify_text(option, str(title))
-        elif option=="Mint Tokens":
-            title = get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title
-            verify_text(option, str(title))         
+    def verify_option_exists(self, list:list):
+        options_list = []
+        options_list.append(get_obj(CommunitySettingsComponents.OVERVIEW_BUTTON.value).title)
+        options_list.append(get_obj(CommunitySettingsComponents.MEMBERS_BUTTON.value).title)
+        options_list.append(get_obj(CommunitySettingsComponents.PERMISSIONS_BUTTON.value).title)
+        options_list.append(get_obj(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value).title)
+        options_list.append(get_obj(CommunitySettingsComponents.AIRDROPS_BUTTON.value).title)
+        
+        # Check if the lists are of equal length
+        if len(options_list) != len(list):
+            return False
+        
+        # Check if the lists have the same elements in the same order
+        for i in range(len(options_list)):
+            if options_list[i] != list[i]:
+                return False
+                         
               
     def select_community_settings_option(self, option:str):
         if option=="Permissions":
             click_obj_by_name(CommunitySettingsComponents.PERMISSIONS_BUTTON.value)
-     
-    def verify_permission_screen_title(self, option:str):
-        if option=="Permissions":
-            title = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_TITLE.value).text
-            verify_text(option, str(title))
+        elif option=="Mint Tokens":
+            click_obj_by_name(CommunitySettingsComponents.MINT_TOKENS_BUTTON.value)
              
-    def verify_welcome_permission_image(self):
-        path = get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_IMAGE.value).source.path
-        verify_text_contains(str(path), "permissions2_3")
+    def verify_welcome_image(self, option:str):
+        path = get_obj(CommunityWelcomeScreenComponents.WELCOME_SCREEN_IMAGE.value).source.path       
+        if option=="Permissions":         
+            verify_text_contains(str(path), "permissions2_3")
+        elif option=="Tokens":
+            verify_text_contains(str(path), "mint2_1")
     
-    def verify_welcome_settings_title(self):
-        verify_equals(get_obj(CommunityPermissionsComponents.WELCOME_SCREEN_SETTINGS_TITLE.value).text, "Permissions")    
+    def verify_welcome_title(self, option:str):
+        title = get_obj(CommunityWelcomeScreenComponents.WELCOME_SCREEN_TITLE.value).text
+        if option=="Permissions":
```

Comment:
@anastasiyaig this string literal used in several places, it will be nice to have some constant for it like `PERMISSIONS_OPTIONS = "Permissions"` cos if it may change in future, it easy to fix it in one place :) The same for others similars literals :)

Reply:
indeed, will do, thank you

Reply:
@borismelnik addressed, mind checking again?